src/utils.rs

```diff
@@ -381,6 +387,12 @@ pub fn sprite_bundle_from_entity_info(tileset: Option<&Handle<Image>>) -> Sprite
     }
 }
 
+#[derive(Bundle, Clone, Debug, Default)]
+pub struct LdtkSpriteSheetBundle {
```

Comment:
Also, I'd like to see some brief docs for it.
```suggestion
/// [`Bundle`] for sprite-sheet-based sprites, similar to bevy 0.13's `SpriteSheetBundle`.
///
/// Implements [`LdtkEntity`], and can be added to an [`LdtkEntity`] bundle with the `#[sprite_sheet_bundle]`
/// field attribute.
/// See [`LdtkEntity#sprite_sheet_bundle`] for attribute macro usage.
/// [`Bundle`]: https://docs.rs/bevy/latest/bevy/ecs/prelude/trait.Bundle.html
#[derive(Bundle, Clone, Debug, Default)]
pub struct LdtkSpriteSheetBundle {
```